src/hooks/protections/serialize_take_off_ped_variation_task.cpp

```diff
@@ -0,0 +1,16 @@
+#include "hooking.hpp"
+#include "util/notify.hpp"
+#include <datanodes/task/ClonedTakeOffPedVariationInfo.hpp>
+
+namespace big
+{
+	void hooks::serialize_take_off_ped_variation_task(ClonedTakeOffPedVariationInfo* info, rage::CSyncDataBase* serializer)
+	{
+		g_hooking->get_original<hooks::serialize_take_off_ped_variation_task>()(info, serializer);
+		if (info->m_parachute_override_model_hash != 0 && info->m_parachute_override_model_hash != RAGE_JOAAT("p_parachute_s"))
```

Comment:
I've had the following false positive hashes for this crash:
 - `185221345`
 - `155602606`
 - `201337717`